README.md

```diff
@@ -88,6 +99,68 @@ In the case where the user has a virtual MFA device already set up they can choo
 This is done by issuing the `--no-mfa` flag on the command line in conjunction with the regular command from
 above.
 
+### setup-new-aws-user add-profile
+
+Before running this tool, you will need to following pieces of information
+
+* IAM role name - This is the IAM Role with permissions allowing access to AWS APIs
+  and services. This is usually something like `admin` or `engineer`. Use the flag
+  `--iam-role` with this value.
+* AWS profile - This is the name of the profile in your `~/.aws/config` profile
+  that you wish to use as the basis for adding new profiles. The `source_profile`
+  and `mfa_serial` is pulled from this profile.
+  Use the flag name `--aws-profile` with this value.
+* AWS Profiles and Account IDs - This is the set of aws profile names you wish to
+  add along with the corresponding AWS account ID. They are referenced as
+  `<AWS_PROFILE>:<AWS_ACCOUNT_ID>`. Use the flag name `--aws-profile-account`
+  with each set you wish to add.
+
+1. Run the setup-new-user script -
+
+    ```sh
+    setup-new-aws-user add-profile \
+      --aws-profile <AWS_PROFILE> \
+      --iam-role <IAM_ROLE> \
+      --aws-profile-account <AWS_PROFILE1>:<AWS_ACCOUNT_ID1> \
+      --aws-profile-account <AWS_PROFILE2>:<AWS_ACCOUNT_ID2>
+    ```
+
+2. Once the tool has completed, you should be able to access the AWS account. You can run the following command filling in the `AWS_PROFILE` value
+
+    ```sh
+    aws-vault exec $AWS_PROFILE -- aws sts get-session
+    ```
+
+#### How `add-profile` modifies your ~/.aws/config
```

Comment:
Probably should add a note that it will override existing configuration if you supply a profile name that already exists.

Reply:
I added a note at the bottom of this section.